charts/shoot-core/charts/csi-packet/templates/controller.yaml

```diff
@@ -0,0 +1,88 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-packet-pd
+  labels:
+    app: csi-packet-pd
+spec:
+  selector:
+    app: csi-packet-pd
+  ports:
+    - name: dummy
+      port: 12345
+
+---
+
+kind: StatefulSet
+apiVersion: apps/v1beta1
+metadata:
+  name: csi-packet-controller
+  namespace: kube-system
+spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      partition: 0
+  serviceName: "csi-packet-pd"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-packet-pd-driver
+  template:
+    metadata:
+      labels:
+        app: csi-packet-pd-driver
+    spec:
+      serviceAccount: csi-controller-sa
+      containers:
+        - name: csi-external-provisioner
+          imagePullPolicy: IfNotPresent
+          image: quay.io/k8scsi/csi-provisioner:v1.0.1
+          args:
+            - "--v=5"
+            - "--provisioner=net.packet.csi"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-attacher
+          imagePullPolicy: IfNotPresent
+          image: quay.io/k8scsi/csi-attacher:v1.0.1
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: packet-driver
+          imagePullPolicy: Always
+          image: docker.io/packethost/csi-packet:379e828
```

Comment:
Same like above.

Reply:
Fixing.

Reply:
This probably needs to be set somewhere. Is that [here](https://github.com/gardener/gardener/blob/master/charts/images.yaml), where I set it as part of the instructions?